react/index.d.ts

```diff
@@ -436,98 +436,165 @@ declare namespace React {
         onEncrypted?: ReactEventHandler<T>;
+        onEncryptedCapture?: ReactEventHandler<T>;
         onEnded?: ReactEventHandler<T>;
+        onEndedCapture?: ReactEventHandler<T>;
         onLoadedData?: ReactEventHandler<T>;
+        onLoadedDataCapture?: ReactEventHandler<T>;
         onLoadedMetadata?: ReactEventHandler<T>;
+        onLoadedMetadataCapture?: ReactEventHandler<T>;
         onLoadStart?: ReactEventHandler<T>;
+        onLoadStartCapture?: ReactEventHandler<T>;
         onPause?: ReactEventHandler<T>;
+        onPauseCapture?: ReactEventHandler<T>;
         onPlay?: ReactEventHandler<T>;
+        onPlayCapture?: ReactEventHandler<T>;
         onPlaying?: ReactEventHandler<T>;
+        onPlayingCapture?: ReactEventHandler<T>;
         onProgress?: ReactEventHandler<T>;
+        onProgressCapture?: ReactEventHandler<T>;
         onRateChange?: ReactEventHandler<T>;
+        onRateChangeCapture?: ReactEventHandler<T>;
         onSeeked?: ReactEventHandler<T>;
+        onSeekedCapture?: ReactEventHandler<T>;
         onSeeking?: ReactEventHandler<T>;
+        onSeekingCapture?: ReactEventHandler<T>;
         onStalled?: ReactEventHandler<T>;
+        onStalledCapture?: ReactEventHandler<T>;
         onSuspend?: ReactEventHandler<T>;
+        onSuspendCapture?: ReactEventHandler<T>;
         onTimeUpdate?: ReactEventHandler<T>;
+        onTimeUpdateCapture?: ReactEventHandler<T>;
         onVolumeChange?: ReactEventHandler<T>;
+        onVolumeChangeCapture?: ReactEventHandler<T>;
         onWaiting?: ReactEventHandler<T>;
+        onWaitingCapture?: ReactEventHandler<T>;
 
         // MouseEvents
         onClick?: MouseEventHandler<T>;
+        onClickCapture?: MouseEventHandler<T>;
         onContextMenu?: MouseEventHandler<T>;
+        onContextMenuCapture?: MouseEventHandler<T>;
         onDoubleClick?: MouseEventHandler<T>;
+        onDoubleClickCapture?: MouseEventHandler<T>;
         onDrag?: DragEventHandler<T>;
+        onDragCapture?: DragEventHandler<T>;
         onDragEnd?: DragEventHandler<T>;
+        onDragEndCapture?: DragEventHandler<T>;
         onDragEnter?: DragEventHandler<T>;
+        onDragEnterCapture?: DragEventHandler<T>;
         onDragExit?: DragEventHandler<T>;
+        onDragExitCapture?: DragEventHandler<T>;
         onDragLeave?: DragEventHandler<T>;
+        onDragLeaveCapture?: DragEventHandler<T>;
         onDragOver?: DragEventHandler<T>;
+        onDragOverCapture?: DragEventHandler<T>;
         onDragStart?: DragEventHandler<T>;
+        onDragStartCapture?: DragEventHandler<T>;
         onDrop?: DragEventHandler<T>;
+        onDropCapture?: DragEventHandler<T>;
         onMouseDown?: MouseEventHandler<T>;
+        onMouseDownCapture?: MouseEventHandler<T>;
         onMouseEnter?: MouseEventHandler<T>;
+        onMouseEnterCapture?: MouseEventHandler<T>;
         onMouseLeave?: MouseEventHandler<T>;
+        onMouseLeaveCapture?: MouseEventHandler<T>;
```

Comment:
`mouseenter` and `mouseleave` events do not have a capture phase. otherwise lgtm!